specificatie/genereervariant/openapi.json

```diff
@@ -400,8 +400,8 @@
       },
       "put" : {
         "tags" : [ "Beheren volgindicaties" ],
-        "summary" : "Toevoegen, wijzigen of beëindigen volgindicatie",
-        "description" : "Toevoegen, wijzigen of beëindigen van een volgindicatie van een specifieke persoon. Wanneer er nog geen volgindicatie is op deze persoon wordt de volgindicatie toegevoegd. Wanneer er al een volgindicatie is op deze persoon, wordt de volgindicatie gewijzigd.\n\nDe einddatum op een volgindicatie kan worden verwijderd (leeg gemaakt) door een leeg object { } te sturen in de request body.\n\nEen volgindicatie kan worden beëindigd door het sturen van einddatum gelijk aan de datum van gisteren.\n",
+        "summary" : "Plaats, wijzig of beëindig een volgindicatie",
+        "description" : "Plaats, wijzig of beëindig een volgindicatie op een specifieke persoon. Als je de persoon nog niet volgt, wordt een volgindicatie geplaatst. Als je de persoon al wel volgt, wordt de volgindicatie gewijzigd.\nJe kunt de einddatum van een volgindicatie verwijderen (leeg maken) door in de request body een leeg object { } te sturen.\nJe kunt een volgindicatie beëindigen door einddatum gelijk aan de datum van vandaag te sturen.\n",
```

Comment:
\n hoort er in?

Reply:
Dit kom ik steeds tegen....

Reply:
'\n' is zeg maar de markdown versie van een html break (<br/>) Alleen wordt daarmee hier een lege regel gegenereerd wat bij html niet het geval is en ik vermoed dat dat ook niet altijd gewenst is. Na elke zin een lege regel lijkt mij wat onzinnig. Een lege regel moet wel een functie hebben. Ik kan ze verwijderen als je dat wil maar soms wordt hierdoor de tekst net even beter leesbaar dus hou dat in het oog. De '\n' aan het eind van een description lijkt mij wat onzinnig maar ik zou moeten kijken wat dat oplevert als ik ze weghaal.

Reply:
Ik zie nu dat deze alleen in de genereervariant staan. Deze '\n' worden dus niet hard ingeklopt maar gegenereerd. Verwijderen kan ik dus niet.